package.json

```diff
@@ -16,6 +16,8 @@
     "babel-core": "6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc"
```

Comment:
It seems like if we're going to update this for 7.x, we might as well change this to `@babel/core` directly as a breaking change.